### PR TITLE
refactor: constants for app.kubernetes.io labels

### PR DESCRIPTION
**Pull request by osmman**

### CODE REVIEW

File: internal/controller/common/utils/kubernetes/common.go [hunk 1]
```diff
@@ -22,25 +22,14 @@ import (
...
-	out := map[string]string{}
-	for key, value := range labels {
-		if key == "app.kubernetes.io/part-of" || key == "app.kubernetes.io/instance" {
-			out[key] = value
-		}
-	}
-	return out
-}
-
 func FilterOutCommonLabels(labels map[string]string) map[string]string {
```
Comment (bouskaJ) [inline internal/controller/common/utils/kubernetes/common.go:27]: I don't think that _kubernetes/common.go_ is correct place to have this function in. We already have similar functions in https://github.com/securesign/secure-sign-operator/blob/main/internal/controller/constants/labels.go#L42 but again - I don't like `constants` package for such functions. Maybe we can extract it to some util package.

File: internal/controller/common/utils/kubernetes/secret.go [hunk 2]
```diff
@@ -68,6 +68,25 @@ func GetSecretData(client client.Client, namespace string, selector *rhtasv1alph
 	return nil, nil
 }
 
+func FindSecrets(ctx context.Context, c client.Client, namespace string, labelSelector string) (*metav1.PartialObjectMetadataList, error) {
```
Comment (bouskaJ) [inline internal/controller/common/utils/kubernetes/secret.go:71]: Please move this function to another PR. It does not have anything common with  `refactor: constants for app.kubernetes.io labels`

File: internal/controller/common/utils/kubernetes/common.go (hunk 1 above)
Comment (osmman) [reply internal/controller/common/utils/kubernetes/common.go:27]: I'm considering moving all the Labels functionality into a dedicated labels package, similar to what we've done for annotations. However, this would involve significant changes during the refactor, so I'd prefer to do it after we've addressed most of our actions to avoid dealing with too many conflicts.

File: internal/controller/common/utils/kubernetes/secret.go (hunk 2 above)
Comment (osmman) [reply internal/controller/common/utils/kubernetes/secret.go:71]: I will remove that
